src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingResourceIntTest.kt

```diff
@@ -760,6 +769,84 @@ class BookingResourceIntTest : ResourceTest() {
       assertThat(response.body!!.alertId).isGreaterThan(1)
       assertThat(response.statusCode).isEqualTo(HttpStatus.CREATED)
     }
+
```

Comment:
no point rewriting all these tests since this endpoint will be removed in the next month when the data moves to DPS